src/components/Card/CardStats.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from "react";
+import styled from "styled-components";
+import { Heading1, Text } from "../../typography";
+
+export interface CardStatsDetails {
+  label: string;
+  value: string;
+}
+
+export interface CardStatsProps {
+  details: CardStatsDetails[];
+}
+
+export const CardStats = ({ ...props }: CardStatsProps): ReactElement => (
+  <CardStatContainer cols={props.details?.length || 0}>
+    {props.details.map((detail) => (
+      <div key={`${detail.label}`}>
```

Comment:
Why do we need to interpolate this?

```suggestion
      <div key={detail.label}>
```

Reply:
Hard for me to see what happened if my comments are closed :(

Reply:
Was auto closed by the `Commit Suggestion` button